dist/src/main/xslt/standalone.xsl

```diff
@@ -34,12 +34,17 @@
         <xsl:attribute name="name">keycloak.import</xsl:attribute>
         <xsl:attribute name="value">&#36;{jboss.home.dir}/standalone/configuration/hawkular-realm.json</xsl:attribute>
       </property>
+      <property>
+        <xsl:attribute name="name">hawkular.events.listener.rest.endpoint</xsl:attribute>
+        <xsl:attribute name="value">http://localhost:8080/hawkular-accounts-events-backend/events</xsl:attribute>
```

Comment:
what about `http://${jboss.bind.address:127.0.0.1}:${jboss.http.port:8080}` instead of `localhost:8080` ?